ruby/example/ThreatExchangeTest.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# ThreatExchange example script.
+require 'ThreatExchange'
+require_relative 'lib/example'
+access_token = 'abc123|somethingsomething'
+TEClient = ThreatExchange::Client.new(access_token)
+#------------------------------------------------------------------------------
+banner('Example Malware Analysis')
+query = {limit: 1000, text: 'bepush', strict_text: false}
```

Comment:
The example script is just showing parameters that can be passed into each method, its not a default nor do you have to set it. So yes it is an optional parameter ;)